src/App.tsx

```diff
@@ -17,18 +62,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedFilter={selectedFilter}
+                searchQuery={searchQuery}
+                onSelectedFilter={(option) => setSelectedFilter(option)}
+                onSearchQuery={(title) => setSearchQuery(title)}
+              />
```

Comment:
```suggestion
                onSelectedFilter={setSelectedFilter}
                onSearchQuery={setSearchQuery}
              />
```

Don't use inliners
You have two options:

1. pass setState callback, this function never changes, so you don't have useless re-renders.
2. create a specific function that takes value and triggers setState function with that value. But you need to wrap 
    this function with useCallback (and you will get render optimization) - that is what you actually did and it's okay
    If you pass (value) => setState(value)- it will cause re-renders of the TodoList